### PR TITLE
[DDW-698] Introduce external links icon

### PR DESCRIPTION
**Pull request by tomothespian**
This PR introduces external links icon.

## Screenshots

_1 SupportSettings_
1  SupportSettings

_2 WalletScreens -> Transactions_
2  WalletScreens -  Transactions

_3 PaperWallets -> Instructions_
3  PaperWallets -  Instructions

_4  PaperWallets -  Completion and link to the address in Cardano Explorer_
4  PaperWallets -  Completion and link to the address in Cardano Explorer

_5  Staking -  Rewards_
5  Staking -  Rewards

_6  Staking -  DelegationStepsIntroDialog_
6  Staking -  DelegationStepsIntroDialog

_7 BlockConsolidationStatus -  Default_
7 BlockConsolidationStatus -  Default

_8. SYSTEM TIME OVERLAY_
_8.1  SystemTimeErrorOverlay - NTP NOT reachable_
8  SystemTimeErrorOverlay - NTP NOT reachable

8.2  SystemTimeErrorOverlay - NTP reachable
8  SystemTimeErrorOverlay - NTP reachable

_9  ManualUpdateOverlay -  Default_
9  ManualUpdateOverlay -  Default

_10  Loading (Report issue) -  Default_
10  Loading (Report issue) -  Default

_11  About_
11  About

_12 Staking - Stake pools  (tooltip)_
12 Staking - Stake pools  (tooltip)

_13 Staking -> Staking Countdown_
Staking -  Staking Countdown

_14 Staking -> Info_
Staking -  Info

_15 Daedalus Diagnostic_
Daedalus Diagnostic

---

## Testing Checklist

- [Slack QA thread](https://input-output-rnd.slack.com/archives/GGKFXSKC6/p1562221685222700)

_Available in storybook:_
- [x] Test Storybook story under: WalletScreens -> Transactions
- [x] Test Storybook story under: PaperWallets -> Instructions
- [x] Test Storybook story under: PaperWallets -> Completion and link to the address in Cardano Explorer
- [x] Test Storybook story under: Staking -> Rewards
- [x] Test Storybook story under: Staking -> DelegationStepsIntroDialog
- [x] Test Storybook story under: BlockConsolidationStatus -> Default
- [x] Test Storybook story under: ManualUpdateOverlay -> Default
**-- additional --**
- [x] Test Storybook story under: Staking - Stake pools 
- [x] Staking -> Staking Countdown
- [x] Staking -> Info

_In-app testing:_
- [ ] SupportSettings
- [ ] SystemTimeErrorOverlay
- [ ] Loading (Report issue) -> Default
- [ ] About
- [ ] Daedalus Diagnostics _(from app menu)_

---

### Basics

- [ ] PR has been assigned and has appropriate labels (`feature`/`bug`/`chore`, `release-x.x.x`)
- [ ] PR is updated to the most recent version of the target branch (and there are no conflicts)
- [ ] PR has a good description that summarizes all changes and shows some screenshots or animated GIFs of important UI changes
- [ ] CHANGELOG entry has been added to the top of the appropriate section (*Features*, *Fixes*, *Chores*) and is linked to the correct PR on GitHub
- [ ] Automated tests: All acceptance and unit tests are passing (`yarn test`)
- [ ] Manual tests (minimum tests should cover newly added feature/fix): App works correctly in *development* build (`yarn dev`)
- [ ] Manual tests (minimum tests should cover newly added feature/fix): App works correctly in *production* build (`yarn package` / CI builds)
- [ ] There are no *flow* errors or warnings (`yarn flow:test`)
- [ ] There are no *lint* errors or warnings (`yarn lint`)
- [ ] There are no *prettier* errors or warnings (`yarn prettier:check`)
- [ ] There are no missing translations (running `yarn manage:translations` produces no changes)
- [ ] Text changes are proofread and approved (Jane Wild / Amy Reeve)
- [ ] Japanese text changes are proofread and approved (Junko Oda)
- [ ] UI changes look good in all themes (Alexander Rukin)
- [ ] Storybook works and no stories are broken (`yarn storybook`)
- [ ] In case of dependency changes `yarn.lock` file is updated

### Code Quality
- [ ] Important parts of the code are properly commented and documented
- [ ] Code is properly typed with flow
- [ ] React components are split-up enough to avoid unnecessary re-renderings
- [ ] Any code that only works in main process is neatly separated from components

### Testing
- [ ] New feature/change is covered by acceptance tests
- [ ] New feature/change is manually tested and approved by QA team
- [ ] All existing acceptance tests are still up-to-date
- [ ] New feature/change is covered by Daedalus Testing scenario
- [ ] All existing Daedalus Testing scenarios are still up-to-date

### After Review
- [ ] Merge the PR
- [ ] Delete the source branch
- [ ] Move the ticket to `done` column on the YouTrack board
- [ ] Update Slack QA thread by marking it with a green checkmark

### CODE REVIEW

**Comment by IuliaDolishniak** [comment]
@tomothespian I wold have a space here
screen

**Comment by IuliaDolishniak** [comment]
@tomothespian I have noticed that on all screens sign is shown after the text but on this screen it is shown before
screen

**Comment by IuliaDolishniak** [comment]
I would add space for the first sign and the second one I think shouldn't be splitted from text
screen

**Comment by tomothespian** [comment]
> @tomothespian I wold have a space here
> screen

Better ?
Screen Shot 2019-07-04 at 14 05 40

**Comment by tomothespian** [comment]
@IuliaDolishniak All requested changes are done. Can you please check?

**Comment by a-rukin** [comment]
image
I think it's good to have this icon near all links always visible -> it will help to recognize what is just an info and what can be clicked

**Comment by tomothespian** [comment]
> image
> 
> I think it's good to have this icon near all links always visible -> it will help to recognize what is just an info and what can be clicked

I will improve that, thanks @a-rukin

**Comment by a-rukin** [comment]
image
"14" is broken for me 😓

**Comment by a-rukin** [comment]
image
this too

**Comment by tomothespian** [comment]
@a-rukin Can I fixed what you requested + arrow position on epochs screen. Can you please re-check.

**Comment by gnpf** [comment]
@tomothespian I noticed that the proportion between the size of the icon and the text is different in all buttons, can this be adjusted?

image

File: source/renderer/app/components/wallet/paper-wallet-certificate/CompletionDialog.js [hunk 1]
```diff
@@ -168,6 +169,7 @@ export default class CompletionDialog extends Component<Props, State> {
                 aria-hidden
               >
                 {cardanoExplorerLink}
+                <SVGInline svg={externalLinkIcon} />
```
Comment (daniloprates) [inline source/renderer/app/components/wallet/paper-wallet-certificate/CompletionDialog.js]: Shouldn't it be underlined only on hovering?

File: source/renderer/app/components/wallet/transactions/Transaction.js [hunk 2]
```diff
@@ -300,6 +301,7 @@ export default class Transaction extends Component<Props> {
                   )}
                 >
                   {address}
+                  <SVGInline svg={externalLinkIcon} />
```
Comment (daniloprates) [inline source/renderer/app/components/wallet/transactions/Transaction.js]: The icon is not underlined, when hovering

File: source/renderer/app/components/wallet/paper-wallet-certificate/InstructionsDialog.js [hunk 3]
```diff
@@ -169,6 +171,7 @@ export default class InstructionsDialog extends Component<Props> {
         aria-hidden
       >
         {intl.formatMessage(messages.cardanoExplorer)}
+        <SVGInline svg={externalLinkIcon} />
```
Comment (daniloprates) [inline source/renderer/app/components/wallet/paper-wallet-certificate/InstructionsDialog.js]: Same, shouldn't it be underlined only on hovering?

File: source/renderer/app/components/wallet/paper-wallet-certificate/CompletionDialog.js (hunk 1 above)
Comment (daniloprates) [reply source/renderer/app/components/wallet/paper-wallet-certificate/CompletionDialog.js]: cc @a-rukin 

image

File: source/renderer/app/components/wallet/paper-wallet-certificate/InstructionsDialog.js (hunk 3 above)
Comment (daniloprates) [reply source/renderer/app/components/wallet/paper-wallet-certificate/InstructionsDialog.js]: cc @a-rukin 

image

File: source/renderer/app/components/wallet/transactions/Transaction.js (hunk 2 above)
Comment (tomothespian) [reply source/renderer/app/components/wallet/transactions/Transaction.js]: @daniloprates it is fixed, nice catch ;)

**Comment by tomothespian** [comment]
> @tomothespian I noticed that the proportion between the size of the icon and the text is different in all buttons, can this be adjusted?
> 
> image

@gnpf it is improved, can you re-check please?

**Comment by gnpf** [comment]
> > @tomothespian I noticed that the proportion between the size of the icon and the text is different in all buttons, can this be adjusted?
> > image
> 
> @gnpf it is improved, can you re-check please?

@tomothespian Looks good!

**Comment by tomothespian** [comment]
@DmitriiGaico External link icon added to Daedalus Diagnostic. Can you take a look please?

**Pull request merged**